README.md

```diff
@@ -343,6 +344,37 @@ app.append().then(() => {
 });
 ```
 
+### Internationalization
+
+Widgets can be internationalized by mixing in `dojo-widgets/mixins/createI18nMixin`. [Message bundles](https://github.com/dojo/i18n) are added by specifying a `bundles` array to the widget factory prototype. When manually accessing localized messages, the bundle should first be passed to `localizeBundle`. If the bundle supports the widget's current locale, but those locale-specific messages have not yet been loaded, then the default messages are returned and the widget will be invalidated once the locale-specific messages have been loaded. Each widget can have its own locale by setting its `state.locale`; if no locale is set, then the default locale as set by [`dojo-i18n`](https://github.com/dojo/i18n) is assumed.
+
+```typescript
+const createI18nWidget = createWidgetBase
```

Comment:
Can we add an example where we use this in `getChildrenNodes` and pass the `i18n` values to a child widget via the state?

something like 
```ts
getChildrenNodes: function() {
    const messages = this.localizedBundle(greetings);

    return [
        d(createButton, { state: { label: messages.buttonLabel } } );
    ];
}
```